index.md

```diff
@@ -21,10 +21,10 @@ A chrultrabook is a modified Chromebook designed to run Windows, Linux, or even
 
 **The docs focuses on two main things:**
 
-*   Installing altOS on your Chromebook.
+*   Installing an altOS on your Chromebook.
 *   Learning in-depth information about your Chromebook.
 
-Because of this, you will be expected to read, learn, and even use Google. This is not a one click procedure.
+Because of this, you **are** expected to read, learn, and use Google. This is not a one click procedure.
```

Comment:
This is my opinion (check with others before making this a reality) but I think we should either just bold this entire line or also bold "This is not a one click procedure"

Reply:
bold